### DISCUSSION TITLE
Doubt regarding the process of subset identification achieved from retrieved data using the metadata

### DISCUSSION THREAD

**Discussion by yuvraj-wale** [Emory-HITI/Niffler#393] [Q&A] [answered]
Hi there, I was working on my proposal for the "Creating shareable "albums" from Niffler data sets" project, and I have a question. On the list of ideas page for this particular project, it states 
`"To use this data, researchers must identify certain subsets of data. This can be achieved by querying the retrieved data. For instance, Niffler stores the metadata of the data retrieved in real-time in a Mongo database. By querying the metadata, subsets of images can be identified." `
- Could you please elaborate on the process of how these subsets of images are identified from the retrieved data using the metadata, which is stored in local environments such as a research server or a cluster where Niffler is being run for more context?

Thanks.
@pradeeban @anbhimi @haritrivedi @judywawira

**Comment by anbhimi** [comment, marked as answer]
As of now, metadata is extracted from DICOM Images using a meta-extraction module from Niffler. The extracted metadata is stored in MongoDB. The metadata is then loaded onto a required platform such as MongoDB Studio or a data frame in pandas. The required images were filtered based on metadata tags and other filterable columns.

**Reply by yuvraj-wale** [reply]
I understand now, thank you for providing clarity on the matter.
